tests/SiteSummaryExtensionTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+use BringYourOwnIdeas\SecurityChecker\Tests\Stubs\SiteSummaryAlertStub;
+
+class SiteSummaryExtensionTest extends SapphireTest
+{
+    protected static $fixture_file = 'PackagesWithAlerts.yml';
+
+    protected $requiredExtensions = [
+        SiteSummary::class => [SiteSummaryAlertStub::class]
+    ];
+
+    public function setUp()
+    {
+        parent::setUp();
+        $manifest = new SS_ConfigManifest(BASE_PATH, true);
+        $moduleDoesNotExist = $manifest->moduleExists('bringyourownideas/silverstripe-maintenance');
+        if ($moduleDoesNotExist) {
+            $this->markTestSkipped(
+                'Module bringyourownideas/silverstripe-maintenance is required for this test, but is not present.'
+            );
+        }
```

Comment:
See comments in previous test class. Is three copies of this comment enough of a reason to make an AbstractMaintenanceTest or something? 😆

Reply:
I did ponder it - but also thought this very comment (use `class_exists` instead) might come up, so thought to not spend too much time on it :P
I don't think a simple class_exists call warrants an abstract superclass.

Reply:
It's an example of adding maintenance burden, since you need to update the logic in three places before it's even been merged 😄 if you want to leave it there thats fine too